build/config/c++/c++.gni

```diff
@@ -43,6 +43,12 @@ use_custom_libcxx =
     use_custom_libcxx || (use_custom_libcxx_for_host && !is_a_target_toolchain)
 use_custom_libcxx = use_custom_libcxx && !is_nacl
 
+# Cobalt comment: Override the use_custom_libcxx variable for partner toolchains to
+# avoid linking with chromium's C++ libraries.
+use_custom_libcxx =
```

Comment:
Seems fine, though we should put this whole thing in a conditional block (and maybe that means you don't need the `is_starboard` check in the value